packages/hosted-form-v2/src/hosted-field-type.ts

```diff
@@ -4,6 +4,7 @@ enum HostedFieldType {
     CardName = 'cardName',
     CardNumber = 'cardNumber',
     Note = 'note',
+    Hidden = 'hidden',
```

Comment:
What does this do?

Reply:
Answered here https://github.com/bigcommerce/checkout-sdk-js/pull/2743/files#r1861517111